### DISCUSSION TITLE
What is really behind the Amnuts source code...

### DISCUSSION THREAD

**Discussion by amnuts** [amnuts/Amnuts#14] [General]
![the real amnuts code](https://user-images.githubusercontent.com/684421/219978467-c274782e-f3e4-46e4-a1d1-c80ae7e67454.png)

**Comment by Uzume** [comment]
And here I thought Silence was behind it (I really hope she doesn't look like that). Of course I doubt many have referred to her by her handle in a very long time too (does she still use that online anywhere?).
